### DISCUSSION TITLE
Octoprint will not connect

### DISCUSSION THREAD

**Discussion by manningcustom** [feelfreelinux/octo4a#157] [General]
I am trying to use octoprint4a.   I have it running on my tablet.   The server is running and it says the printer is connected.   But when I go to octoprint in my browser it will not connect and just  times out.   I have switched from auto to manual and tried every listed baudrate.   no luck

What am I missing?

**Comment by Chris2388** [comment]
Im having the same issues with a Galaxy S6 fresh reset. Tried 1.0.4 -to newest release all the same issues.  Tried 3 different WIFI connections
